pkg/agent/openflow/fields.go

```diff
@@ -137,6 +139,16 @@ var (
 	// reg7(NXM_NX_REG7)
 	// Field to store the GroupID corresponding to the Service.
 	ServiceGroupIDField = binding.NewRegField(7, 0, 31, "ServiceGroupID")
+
+	// reg8(NXM_NX_REG8)
+	// Field to store the VLAN ID.
+	VLANIDField = binding.NewRegField(8, 0, 11, "VLANID")
+	// Field to store the CtZone type.
+	CtZoneTypeField       = binding.NewRegField(8, 12, 15, "CtZoneType")
+	IPCtZoneTypeRegMark   = binding.NewRegMark(CtZoneTypeField, 0b0001)
+	IPv6CtZoneTypeRegMark = binding.NewRegMark(CtZoneTypeField, 0b0011)
+	// Field to store the CtZone ID, which is a combination of VLANIDField and CtZoneTypeField
+	CtZoneField = binding.NewRegField(8, 0, 15, "CtZoneID")
```

Comment:
I think you want to use this field to cache VLAN ID,  and then use the field to create/access ct_zone value, is it possible to use filed `OXM_FIELD_VLAN_VID` directly?

Reply:
No. `OXM_FIELD_VLAN_VID` doesn't exist if the in_port is access port and we should not set it manually because it will break the NORMAL action.
Meanwhile, we also need bit 12 to 15 to store other information such as the IP protocol, and the whole 16 bits are used for CtZone